config/vufind/export.ini

```diff
@@ -71,4 +71,5 @@ headers[] = "Content-type: application/x-bibtex; charset=utf-8"
 
 [RIS]
 requiredMethods[] = getTitle
-headers[] = "Content-type: application/x-research-info-systems; charset=utf-8"
\ No newline at end of file
+headers[] = "Content-type: application/x-research-info-systems; charset=utf-8"
+label = Citavi/RIS
```

Comment:
Does it make sense to label this with a specific product name by default, or should we leave this as a local customization? (I assume that Citavi is just one of several tools that can handle RIS, and this may or may not make sense depending on the location of the VuFind installation).